compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeContainer.uikit.kt

```diff
@@ -313,7 +313,7 @@ internal class ComposeContainer(
         )
     }
 
-    private fun setContent(content: @Composable () -> Unit) {
+    private fun createMediatorIfNeeded() {
         val mediator = mediator ?: ComposeSceneMediator(
```

Comment:
Let's to not overuse `also`/etc helpers. In this particular case, the logic became less obvious and more error prone because of mixing `?:` /`this`/`it`.

To make it more readable, I would extract `createMediator`:
```
    private fun createMediatorIfNeeded() {
       this.mediator = mediator ?: createMediator()
       mediator.setLayout(SceneLayout.UseConstraintsToFillContainer)
    }

    private fun createMediator() = ComposeSceneMediator(
        container = view,
        configuration = configuration,
        focusStack = focusStack,
        windowContext = windowContext,
        coroutineContext = coroutineDispatcher,
        renderingUIViewFactory = ::createSkikoUIView,
        composeSceneFactory = ::createComposeScene,
    ).also {
        it.setContent {
            ProvideContainerCompositionLocals(this, content)
        }
    }
}
```

Reply:
Makes sense, actually doing it also revealed that we don't need to call `setLayout` multiple times in this specific case.